### PR TITLE
Adjusted OnRunJavaScriptAlertPanel according to the WKUIDelegate recipe

### PR DESCRIPTION
**Pull request by DevenColletteNV**
## PR Type
What kind of change does this PR introduce?

Bugfix 

## What is the current behavior?
Crash when OnRunJavaScriptAlertPanel is called 

## What is the new behavior?
No Crash

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by MatFillion** [comment]
@DevenColletteNV will validate with larger analytics if this really fixes the crash he is looking into. And create the appropriate issue.  I added label to not merge yet.

File: src/Uno.UI/UI/Xaml/Controls/WebView/UnoWKWebView.iOS.cs [hunk 1]
```diff
@@ -188,7 +188,9 @@ private void OnRunJavaScriptAlertPanel(WKWebView webview, string message, WKFram
 			var alert = UIKit.UIAlertController.Create(string.Empty, message, UIKit.UIAlertControllerStyle.Alert);
 			alert.AddAction(UIKit.UIAlertAction.Create(OkString, UIKit.UIAlertActionStyle.Default, null));
 
-			controller?.PresentViewController(alert, true, completionHandler);
+			controller?.PresentViewController(alert, true, null);
+
+			completionHandler();
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/Controls/WebView/UnoWKWebView.iOS.cs]: Why ? :)

Comment (carldebilly) [reply src/Uno.UI/UI/Xaml/Controls/WebView/UnoWKWebView.iOS.cs]: The completition must be called sooner.  The fix is correct.

**Comment by davidjohnoliver** [comment]
should be linked with an issue

**Pull request merged**